subxt/src/runtime_api/runtime_types.rs

```diff
@@ -42,11 +42,16 @@ where
         // Ensure that the returned future doesn't have a lifetime tied to api.runtime_api(),
         // which is a temporary thing we'll be throwing away quickly:
         async move {
-            let data = client
+            let call_parameters = call_parameters.unwrap_or_default();
+            let hex_encoded_call_params = format!("0x{}", hex::encode(call_parameters));
+            let bytes: types::Bytes = client
                 .rpc()
-                .state_call(function, call_parameters, Some(block_hash))
+                .request(
+                    "state_call",
```

Comment:
Yeah; let's just make the result of this `call_raw` be `Res: Decode` :)

Reply:
@lexnv Good idea. Should I change something in the code now, or is that a thing for another PR? 
@jsdw I would like to make your suggested change. But the function name should stay `call_raw`? So the raw then only refers to the `call_parameters: Option<&'a [u8]>` and not to the return type anymore, right?

Reply:
Yeah, keep the `call_raw`; this is because in the future we'll be adding a much more ergonomic `call` method which will be the preferred way to use this interface :)